tests/test_settings.py

```diff
@@ -0,0 +1,42 @@
+import importlib
+import typing
+
+import pytest
+
+import microbootstrap.settings
+
+
+@pytest.fixture
```

Comment:
maybe move it to fixtures?

Reply:
I hope we can keep mangling with environment local to this file. If you still think it’s a good idea to move it there I’ll do it alright :)